flake.nix

```diff
@@ -22,20 +22,20 @@
     #   - https://discourse.nixos.org/t/differences-between-nix-channels/13998
     # How to update the revision
     #   - `nix flake update --commit-lock-file` # https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-flake-update.html
-    nixpkgs.url = "https://channels.nixos.org/nixos-25.05/nixexprs.tar.xz";
+    nixpkgs.url = "github:NixOS/nixpkgs/release-25.11"; # TODO: Use https://channels.nixos.org/nixos-25.11/nixexprs.tar.xz after the branch off. release-25.11 is not available in channels.nixos.org
```

Comment:
Using GitHub as a channel will make rate limit error such as https://github.com/kachick/dotfiles/issues/1262. Should I temporally drop the container workflow? Keeping this PR longer is annoy to me 🤔